### PR TITLE
MINOR: make controller helper methods private

### PR DESCRIPTION
**Pull request by onurkaraman**

### CODE REVIEW

**Comment by asfgit** [comment]
SUCCESS 
 8057 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8057 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
FAILURE
 7949 tests run, 5 skipped, 1 failed.
--none--

File: core/src/main/scala/kafka/controller/KafkaController.scala [hunk 1]
```diff
@@ -124,13 +124,7 @@ class KafkaController(val config: KafkaConfig, zkClient: KafkaZkClient, time: Ti
 
   def epoch: Int = controllerContext.epoch
 
-  def state: ControllerState = eventManager.state
-
-  def clientId: String = {
```
Comment (viktorsomogyi) [inline core/src/main/scala/kafka/controller/KafkaController.scala]: Why do you need to remove this?

**Review by viktorsomogyi — COMMENTED** [review]
Hi Onur, I reviewed your PR if you don't mind and made a comment :).

File: core/src/main/scala/kafka/controller/KafkaController.scala (hunk 1 above)
Comment (onurkaraman) [reply core/src/main/scala/kafka/controller/KafkaController.scala]: `def clientId` is never used.

**Review by ijuma — COMMENTED** [review]
Thanks for the PR. Since you are doing this, maybe we can take the chance to group all public methods and group all private ones? Also, there are some methods in the middle of the events that should probably be moved higher.

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM. Feel free to merge after the tests pass.

**Comment by asfgit** [comment]
SUCCESS 
 8065 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8065 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8065 tests run, 5 skipped, 0 failed.
--none--

**Pull request closed without merging**